modules/stochastic_tools/doc/content/modules/stochastic_tools/examples/libtorch_drl_control.md

```diff
@@ -0,0 +1,165 @@
+# Deep Reinforcement Learning Control Using Libtorch
```

Comment:
I prefer not to. My opinion is that we should only use that function there is additional information that is missing because it needs MOOSE to be compiled with libtorch.

Reply:
Now that I come to think of it, I will do it.